package.json

```diff
@@ -48,6 +48,11 @@
     "exclude": [
       "dist/**/*.spec.js",
       "dist/**/*.d.ts"
+    ],
+    "all": true,
```

Comment:
How does `all` work with `include`/`exclude`? Is it needed since all files will match based on those two glob arrays?